Terminal.Gui/View/Adornment/Adornment.cs

```diff
@@ -37,6 +37,11 @@ public override Rect Bounds
         set => throw new InvalidOperationException ("It makes no sense to set Bounds of a Thickness.");
     }
 
+    /// <summary>
+    ///     Gets the rectangle that describes the inner area of the Adornment. The Location is always (0,0).
+    /// </summary>
+    public override Rect ContentArea => Thickness?.GetInside (new Rect (Point.Empty, Frame.Size)) ?? new Rect (Point.Empty, Frame.Size);
```

Comment:
Because the `Parent.Bounds` may have negative location. Don't forget that this PR now allow using negative bounds location and thus the dimension is also changed. The `ContentArea` is only the available space inside adornments used to set the driver clip area.

Reply:
I'm confused.

Can you please update the API docs for `Bounds` to represent what you've done here?

Reply:
I'm still working on this and I didn't decided well what I'm going to do. I'm testing the `ScrollBar` on `Adornments` and on `ContentArea` but he behaves differently related with dimensions.

Reply:
> I'm still working on this and I didn't decided well what I'm going to do. I'm testing the `ScrollBar` on `Adornments` and on `ContentArea` but he behaves differently related with dimensions.

Ok, I'm going to avoid thinking deeply about `Bounds` v `ContentArea` until you tell me it's time. For me, the best way to gain clarity on things like this is to write the API docs as clearly as possible. I understand you're less comfy about this, but I encourage you to give it your best shot. Happy to riff with you on it if that helps!